node/typings/Checkout.ts

```diff
@@ -371,6 +371,10 @@ interface RatesAndBenefitsData {
     name: string
     featured: boolean
     description: string
+    additionalInfo: {
```

Comment:
Since this can be undefined/null, can you update this type accordingly? This should be enough:

```suggestion
    additionalInfo?: {
```